docs/content/preview/yugabyte-cloud/cloud-examples/hasura-cloud.md

```diff
@@ -51,32 +51,30 @@ To create a project in Hasura Cloud:
 
 1. From the Hasura Cloud Dashboard, under **Projects**, click **New Project**.
 
-1. Select **Free Tier**, leave the default region, and enter a name for your project.
-
-    <br/><br/>
+1. Select **Free Tier**.
 
     ![Create Hasura project](/images/deploy/yugabyte-cloud/hasura-create-project.png)
 
-    <br/><br/>
-
-1. Click **Create project**.
+1. Click **Create Free Project**.
 
-    The project details are displayed. These include the GpaphQL API endpoint URL and Admin Secret, which is used for connecting applications.
+    The project details are displayed. These include the **GpaphQL API** endpoint URL and **Admin Secret**, which are used for connecting applications.
```

Comment:
```suggestion
    The project details are displayed. These include the **GraphQL API** endpoint URL and **Admin Secret**, which are used for connecting applications.
```

Reply:
Also, is it better to mention about copying the Admin secret details which the user can use while deploying an application?